adage/adage/config.py.template

```diff
@@ -7,6 +7,10 @@
 # and replace with values tailored to your deployment.
 import random
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I think if we build a `CIRCLECI_CONFIG` by starting with `CIRCLECI_CONFIG = TEST_CONFIG.copy()` and then putting the @cgreene's changes to `TEST_CONFIG` into that via `CIRCLECI_CONFIG.update({...` (following what I did for `CI_CONFIG`) then all of the `BASE_DIR` and `REPO_ROOT` references, which confuse `fab` deployment, will be isolated where they are needed.

With this change, I think we could merge and not have any conflicts with the existing AWS and development deployment `fab` scripts. Does that make sense?